tests/browser-tests/lib/baseUrl.ts

```diff
@@ -0,0 +1,2 @@
+const BASE_URL = process.env.BASE_URL || 'localhost:9000';
+export { BASE_URL };
```

Comment:
We generally put things like this in a utils folder, not a massive issue but probably worth moving.

Reply:
Moved to `test-utils` in https://github.com/libero/reviewer-client/pull/247/commits/8f5f3a2006fd65f6b62d364d9ba53a5f30787478.